torch/fx/graph_module.py

```diff
@@ -215,6 +202,8 @@ def graph(self, g) -> None:
         corresponds to ``g``
         """
         self._graph = g
+        g._gm = self if not g._owners else None
+        g._owners += 1
```

Comment:
It's still weird that `GraphModule` is reaching into private attributes of `Graph`. it's probably better to define a method/property on `Graph` like `register_owner(gm : GraphModule)` that encapsulates this logic. That can also be where the accounting for multiple owners happens